package.json

```diff
@@ -16,7 +16,16 @@
     "redux",
     "javascript"
   ],
-  "author": "Billy Montgomery billy.montgomery@hixme.com",
+  "author": {
+    "name": "Billy Montgomery",
+    "email": "billy.montgomery@gmail.com",
+    "url": "https://github.com/billyxs"
+  },
+  "contributors": [{
+    "name": "Calvin Cieslak",
+    "url": "https://github.com/seesleestak"
+  }],
```

Comment:
😀

Reply:
Feel free to update

Reply:
Looks good to me. Thanks for adding it in :)